requirements/requirements.txt

```diff
@@ -44,79 +44,81 @@ django-taggit==0.22.2
 django==1.11.20
 djangorestframework-yaml==1.0.3
 djangorestframework==3.7.7
+docutils==0.14            # via python-daemon
```

Comment:
We explicitly removed docutils from this list and put a Makefile directive in here https://github.com/ansible/awx/blob/devel/Makefile#L149 to install docutils. At pip install time a dependency of ansible-runner requires docutils to exist to "build" the package .. it's weird. So we pre-install docutils before installing that ansible-runner dep.

Reply:
IIRC, the logic I had for removing the git requirements was that, it turned out that no dependencies pulled it.

Since I originally did that, I've done several rebases of this, running the tool again, and here it is, shows up as a surprise.

It appears that it is listed as a `setup_requires`, but not a `install_requires`.

https://pagure.io/python-daemon/blob/master/f/setup.py

I'm still looking into what we can do about this. My current thinking is to change the instructions to say "remove all the git requirements at the end"

Reply:
Ok, I removed it and added a note.

I still don't think we need to maintain the `-e git+` stuff that we used to have. Maybe we need to add a note to check that no dependencies _conflict_ with the git requirements, but that doesn't mean that we need to throw them into the pot before we cook it.

Reply:
Also, I missed this from docutils

```python
# Docutils is only required for building, but Setuptools can't distinguish
# dependencies properly.
# See <URL:https://github.com/pypa/setuptools/issues/457>.
setup_kwargs['install_requires'].append("docutils")
```

So yeah, either we tell pip-compile to ignore libraries like this, or we just let it run and have them deleted at the end. My preference is switching to the latter here.